specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/virtualWan.json

```diff
@@ -5801,6 +6042,123 @@
         }
       }
     },
+    "VpnNatRuleMapping": {
+      "properties": {
+        "addressSpace": {
+          "type": "string",
+          "description": "Address space for Vpn NatRule mapping."
+        }
+      },
+      "description": "Vpn NatRule mapping."
+    },
+    "VpnGatewayNatRuleProperties": {
```

Comment:
>VpnGatewayNatRuleProperties [](start = 5, length = 27)

Are all read-only properties marked as read only?

Reply:
yes egressVpnSiteLinkConnections and ingressVpnSiteLinkConnections of VpnGatewayNatRuleProperties are readonly.